### PR TITLE
Refactor intake, pivot, and shooter subsystems

### PR DESCRIPTION
**Pull request by boomermath**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 1]
```diff
@@ -57,7 +45,19 @@ public boolean getBeamBreak() {
...
+    public Command rollIn(double vel) {
+        // TODO: add some kind of warning logging if a negative value is passed
+
+        return roll(Math.abs(vel));
+    }
+
+    public Command rollOut(double vel) {
+        // TODO: add some kind of warning logging if a positive value is passed
+
+        return roll(-Math.abs(vel));
```
[the remark this answers is not in the document]
Comment (samperlmutter) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java:61]: Make sure to use the javadoc syntax so that it appears in autocomplete suggestions

**Comment by drew-beamer** [comment]
Let's resolve merge conflicts and merge this. The PR is already pretty massive.

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 2]
```diff
@@ -5,25 +5,22 @@
...
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 import frc.robot.constants.IntakeConstants;
 import frc.robot.constants.TalonFXConstants;
 
 public class IntakeSubsystem extends SubsystemBase {
     private final TalonFX intakeKraken;
-
     private final DigitalInput beamBreak;
-    public final Trigger intakeOccupiedTrigger;
```
Comment (drew-beamer) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java:17]: why did we get rid of the trigger?

**Comment by drew-beamer** [comment]
Looks good to me as far as I can tell. I just want to do one more on-bot test to verify we haven't broken anything.

**Review by drew-beamer — APPROVED** [review]

**Pull request merged**